### PR TITLE
[DOCS] Adds query strings and headers to all full code examples in GX Core docs

### PR DESCRIPTION
**Pull request by Rachel-Reverie**
## Description
Permits linking directly to full example code for any given example in the GX Core docs:
- Adds query strings to all procedure/sample_code tab groups in the GX Core docs.
- Adds headers above all procedure/sample_code tab groups in the GX Core docs.

## Definition of done
- [X] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [X] Appropriate tests and docs have been updated

### CODE REVIEW

**Comment by rachhouse** [comment]
@Rachel-Reverie This is a nice docs feature add!

I see that Try GX Core isn't included in the updates, I assume we do want to standardize tab headers and add the query params capability to that page as well? Currently the tab names for the two Try GX workflows are "Walkthrough" and "Full example code" - those can be updated to "Instructions" and "Sample code" to maintain consistency.

**Comment by Rachel-Reverie** [comment]
> @Rachel-Reverie This is a nice docs feature add!
> 
> I see that Try GX Core isn't included in the updates, I assume we do want to standardize tab headers and add the query params capability to that page as well? Currently the tab names for the two Try GX workflows are "Walkthrough" and "Full example code" - those can be updated to "Instructions" and "Sample code" to maintain consistency.

File: docs/docusaurus/docs/core/_create_expectations/expectation_suites/manage_expectation_suites.md [hunk 1]
```diff
@@ -24,9 +24,16 @@ An Expectation Suite contains a group of Expectations that describe the same set
 
 ## Create an Expectation Suite
 
-<Tabs>
+<Tabs 
+   queryString="procedure"
+   defaultValue="instructions"
+   values={[
```
Comment (tyler-hoffman) [inline docs/docusaurus/docs/core/_create_expectations/expectation_suites/manage_expectation_suites.md:30]: Are you required to pass in `values` if you pass in the other parameters? Feels tedious!

**Review by tyler-hoffman — APPROVED** [review]
This is great! I was actually going to ask for this after linking some folks to docs this morning!

File: docs/docusaurus/docs/core/introduction/try_gx.md [hunk 2]
```diff
@@ -55,9 +55,18 @@ This example workflow walks you through connecting to data in a Pandas DataFrame
 This example requires that [Pandas](https://pandas.pydata.org/) is installed in the same Python environment where you are running GX Core.
 :::
 
-<Tabs>
+### Procedure
```
Comment (rachhouse) [inline docs/docusaurus/docs/core/introduction/try_gx.md:60]: @Rachel-Reverie Small nit, is there any way we can restrict the small rendered nav TOC on the right to only show the H2 headings? The **Procedure** headings look kind of clunky in there, and I see on other pages, like _Connect to SQL data_ the righthand nav only shows the top level sections.

Screenshot 2024-08-23 at 9 42 52 AM

File: docs/docusaurus/docs/core/_create_expectations/expectation_suites/manage_expectation_suites.md (hunk 1 above)
Comment (Rachel-Reverie) [reply docs/docusaurus/docs/core/_create_expectations/expectation_suites/manage_expectation_suites.md:30]: You aren't, but I've been plugging these in using a pycharm live template so it actually is quite quick: I just select the content I want to put in the "Instructions" tab, press a few buttons and whoosh! Instant tab group with header and Instructions/Sample Code tabs.  Add the code block for the full example to the Sample Code tab and done.

File: docs/docusaurus/docs/core/introduction/try_gx.md (hunk 2 above)
Comment (Rachel-Reverie) [reply docs/docusaurus/docs/core/introduction/try_gx.md:60]: We can.  I'll add an edit to restrict those to H2.

**Review by rachhouse — APPROVED** [review]
Looks great, thanks!

**Pull request merged**